angular_token_auth.js

```diff
@@ -297,13 +301,19 @@
             }
         };
 
-        //use cookies if available, otherwise try localstorage
-        if (storageMethods['cookie'].test() === true) {
-            return storageMethods['cookie'];
-        } else if (storageMethods['localStorage'].test()) {
-            return storageMethods['localStorage'];
+        if (storageMethods[MODULE_SETTINGS.STORAGE_METHOD]) {
+            return storageMethods[MODULE_SETTINGS.STORAGE_METHOD];
```

Comment:
Adding the test as you suggest provides some fallback to 'auto' if the specified method does not work. However, the only logical reason for choosing a specific method is if the 'auto' detection is not working for you.